chirp/drivers/iradio_uv_5118plus.py

```diff
@@ -208,7 +208,7 @@ def _enter_programming_mode(radio):
             if ack == CMD_ACK:
                 exito = True
                 break
-        except errors.RadioError:
+        except Exception:
```

Comment:
This one probably doesn't need an `except` around it, since nothing here can really raise an exception, unless ack was something other than a `bytes`. No need to change it now though.